doc/nrf/ug_matter_overview_multi_fabrics.rst

```diff
@@ -12,6 +12,15 @@ Because each Matter fabric has an administrator, this allows the device to be ad
 
 This Matter feature is key for achieving Matter's smart home interoperability.
 
+.. note::
```

Comment:
It's very subjective.. but personally i would prefer to have youtube video embedded only on main documentation page and if anything, link to the webinar/main page from here. I have a feeling we might have too many videos embedded on different pages, and videos tends to become outdated.

Reply:
Let's discuss as part of another task: https://projecttools.nordicsemi.no/jira/browse/KRKNWK-16086 I removed these additions from this PR.